dotnet/src/Autogen.Ollama/Agent/OllamaClientAgent.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// OllamaClientAgent.cs
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoGen.Core;
+
+namespace Autogen.Ollama;
+
+/// <summary>
+/// An agent that can interact with ollama models.
+/// </summary>
+public class OllamaClientAgent : IStreamingAgent
```

Comment:
Maybe consider renameing this class to `OllamaAgent` and remove the actual `OllamaAgent` class? Because there's no `OllamaClient` defined in this package

Reply:
ok, can be renamed yes

Reply:
I've removed the ollama agent class but my idea behind this client is to allow not only chat completion operations.. but for now I keep that version on my computer until new features will arrive